gradebookng/tool/src/webapp/scripts/gradebook-gbgrade-table.js

```diff
@@ -1255,10 +1255,16 @@ GbGradeTable.modelIndexForStudent = function(studentId) {
 };
 
 
-GbGradeTable.colForAssignment = function(assignmentId) {
-  return GbGradeTable.findIndex(GbGradeTable.instance.view.settings.columns, function(column) {
-           return column._data_ && column._data_.assignmentId === parseInt(assignmentId);
-         });
+GbGradeTable.colForAssignment = function(assignmentId, array) {
+    if(array == undefined){
```

Comment:
Need a space after if and use ===

Reply:
Changes done.
The problem occurs with or without sections. Always when one or more items are hidden at the beginning and some others at the end.